iam-login-service/src/main/java/it/infn/mw/iam/config/IamConfig.java

```diff
@@ -49,13 +49,33 @@
 import it.infn.mw.iam.notification.service.resolver.NotifyGmStrategy;
 import it.infn.mw.iam.notification.service.resolver.NotifyGmsAndAdminsStrategy;
 import it.infn.mw.iam.persistence.repository.IamAupRepository;
+import org.springframework.ui.freemarker.FreeMarkerConfigurationFactoryBean;
+import org.springframework.web.servlet.view.freemarker.FreeMarkerViewResolver;
 
 @Configuration
 public class IamConfig {
 
   @Value("${iam.organisation.name}")
   private String iamOrganisationName;
 
+  @Bean
```

Comment:
This configuration is likely handled automatically by the spring boot freemarker starter